middleman_protocol/middleman_protocol/concent_golem_messages/message.py

```diff
@@ -49,6 +58,16 @@ class TransactionAbstractMessage(NonceAbstractMessage):
         'data',
     ] + NonceAbstractMessage.__slots__
 
+    def slots(self) -> List[List[Union[str, bytes, int]]]:
+        slots = super(TransactionAbstractMessage, self).slots()
```

Comment:
Same here, why is it required to convert those values?

Reply:
Answered above https://github.com/golemfactory/concent/pull/1050#discussion_r245930285